whitelabelco/sofo-foods
# Cp/feat/updates to order page

### Description 💻

Resolves: #127 & #177 &
#156 

### Implementation 💡

Requested changes to Order page:
- place order now button
- update hero content
- add content under form title
- add captcha verification
- remove fax from location cards
- make numbers clickable
- on mobile, display Download App button that links to app/play store 

[Comps](https://www.figma.com/design/04vc8B8NFKlryNS0e1h57T/SoFo-Foods?node-id=248-5873&node-type=frame&t=WzKbbgAFIHAEYq0r-0)
#### Type of change


### How to test 🔍

- `npm i`
- `npm run dev`
- navigate to `http://localhost:3000/`
- click "Order Now"
- compare to comps
- test captcha
- submit button should be disabled until form and captcha are complete
- switch to mobile view, confirm app button opens to app linked in #156 

### Screenshots/Demos 📷

### Checklist ✅
